api/external/infra_proxy/request/environments.proto

```diff
@@ -5,11 +5,22 @@ option go_package = "github.com/chef/automate/api/external/infra_proxy/request";
 
 import "google/protobuf/struct.proto";
 
+message SearchQuery {
+	// The search query used to identify a list of items.
+	string q = 1;
+	// The number of result pages to return.
```

Comment:
The PR preamble disagrees, stating `start` is the "offset of the records" which I think is correct and L11 here is incorrect. But what units is `offset`? I am guessing that is "pages" where a page is defined by the `rows` property?
_(This applies to several PRs and several proto files in each PR)._

Reply:
ohh yeah, will match and updated it accordingly.

Also, I tried to match parity with chef server API to named the exact same.  ref https://docs.chef.io/server/api_chef_server/#get-47

Automate uses most of the places `per_page`, `page` variables for pagination. do we need to rename it?

Reply:
Yes, I missed that; good catch. Looking over the existing ones in the rest of the code base, the description for `page` is wrong on most of them.
Let's go with this, please:
```
    // Starting page for the results.
	int32 page = 23;
	// Number of results on each page.
	int32 per_page = 24;
```

Reply:
Thanks for the renames, but you missed my last comment above: the comment on `page` is wrong in a couple places where you have copied it. Needs to be as shown above, "Starting page for the results."